src/components/treeInfo/index.tsx

```diff
@@ -92,7 +92,7 @@ const TreeInfo: React.FC<TreeProps> = ({
           ? 'Check out this tree I adopted!'
           : adopted
           ? 'Check out this tree near you!'
-          : `This tree${
+          : `This ${(siteData.entries[0].commonName ?? 'tree').toLowerCase()}${
```

Comment:
🔧 Could we show the tree name on lines 92 and 94 as well? This will probably require a helper function (preferably with unit tests). Also, we might have to do something like `siteData.entries[0]?.commonName` to account for the case where a site has no site entries

Reply:
I like the use of 'tree' as a default value, but I agree that maybe you can clean this up by defining some local treeName variable that is either siteData.entries[0].commonName or 'tree' when the site has no entries. And then add that variable to lines 92, 94, and 95

Reply:
I like the use of 'tree' as a default value, but I agree that maybe you can clean this up by defining some local treeName variable that is either siteData.entries[0].commonName or 'tree' when the site has no entries. And then add that variable to lines 92, 94, and 95

Reply:
Addressed feedback!